src/subdomains/core/liquidity-management/entities/liquidity-management-rule.entity.ts

```diff
@@ -107,6 +112,19 @@ export class LiquidityManagementRule extends IEntity {
     return this;
   }
 
+  updateRuleSettings(reactivationTime: number | undefined): this {
+    if (reactivationTime !== undefined) this.reactivationTime = reactivationTime;
+
+    return this;
+  }
+
+  shouldReactivate(): boolean {
+    return (
+      this.status === LiquidityManagementRuleStatus.PAUSED &&
+      Util.minutesDiff(this.updated, new Date()) > this.reactivationTime
```

Comment:
when testing locally I've got a time mismatch between DB and local API. Should not be an issue in production, since api and db would be in sync clock

Reply:
Yes, makes sense 👍